src/metabase/data_apps/models.clj

```diff
@@ -169,18 +209,17 @@
 (defn latest-release
   "Get the latest release info for a data app."
   [app-id]
-  ;; app_definition_id is only needed for testing
-  (t2/select-one [:model/DataAppRelease :id :app_definition_id :created_at]
```

Comment:
If you're removing the column filtering here, let's still restrict columns in the API.